README.md

```diff
@@ -0,0 +1,219 @@
+
+## Getting started with binge
+Install binge by running ``yarn global add binge``.
+
+**Note**: binge requires [Yarn](https://yarnpkg.com/lang/en/) and doesn't support npm.
+
+### Repository structure
+Structure your repository as follows:
+
+```
+…
+- client
+    - package1
+        - …
+        - src
+        - .babelrc
+        - .npmignore
+        - package.json
+        - yarn.lock
+    - …
+    - packageN
+        - …
+    - app
+        - src
+        - package.json
+        - …    
+…
+```
+
+Your client folder needs to contain a root-level folder for each local package, plus one additional folder that contains the global entry point to (the *root* of) your app.
+In our example (see: [./examples/watch-app](./examples/watch-app)), we name this folder ``app``.
+
+**Note:** binge supports nested local packages. However, we recommend you don't nest your packages.
+This reduces complexity and allow a clearer overview.
+
+Local packages have the same structure as ordinary npm packages, except that they are **not versioned**. 
+
+The ``package.json`` in the root folder lists the local packages as dependencies:
+
+```json
+{
+  "name": "app",
+  "dependencies": {
+      "i-filter-stuff": "file:../i-filter-stuff",
+      "i-provide-text": "file:../i-provide-text"
+  },
+  …
+}
+```
+
+To build the app, navigate to the ``app`` directory and run ``binge bootstrap``.
+binge now installs the dependencies in the local packages, builds the local packages and deploys them and their dependencies to the ``node_modules`` folder of the ``app`` directory. At that, binge moves shared dependencies up the dependency tree to avoid unnecessary duplication of code.
+
+To run the app and watch the source files, run ``binge watch`` in the ``app`` directory.
+
+## Reference documentation
+This section lists and explains all binge commands.
+You find minimal example apps that illustrate the commands at [./examples](./examples).
+
+* ``bootstrap``:
+
+    Installs, builds and deploys the local package tree.
+    For example, running ``binge bootstrap`` in [./examples/bootstrap/root](./examples/bootstrap/root):
+    * installs the dependencies of the three local packages ``i-filter-stuff``, ``i-print-stuff`` and ``i-prodivde-colors``,
+    * builds the three local packages,
+    * deploys the local packages and their dependencies to the ``node_modules`` folder of the ``root`` directory. At that, binge moves shared dependencies up the dependency tree to avoid unnecessary duplication of code.
+
+    **Note:** ``bootstrap`` also runs ``check`` (see below) and fails if ``check`` fails.
+
+* ``check``:
+
+    Checks if the [yarn-lock.json](https://yarnpkg.com/lang/en/docs/yarn-lock/) and ``package.json`` files in the local package tree are in sync. This helps ensure your dependency versions are consistent across machines.
+    For example, running ``binge check`` in [./examples/check-sync/root](./examples/check-sync/root) returns:
```

Comment:
"This helps ensure your dependency versions are consistent across machines."
Replace with:
Lock files ensure consistent build across machines. This command helps ensure that no developer forgot to push lock files changes.